images/macos/macos-11-Readme.md

```diff
@@ -49,8 +48,8 @@
 - Pip 20.3.4 (python 2.7)
 - Pip3 23.3.1 (python 3.12)
 - Pipx 1.3.3
-- RubyGems 3.4.22
-- Vcpkg 2023 (build from commit f56238700)
+- RubyGems 3.4.10
```

Comment:
Due to default ruby being 2.7:
==> vsphere-clone: 	rubygems-update-3.5.1 requires Ruby version >= 3.0.0. The current ruby version is 2.7.8